app/models/shipit/stack.rb

```diff
@@ -251,42 +251,24 @@ def lock_reverted_commits!
       end
     end
 
-    def active_commits
-      return [] unless active_task?
-
-      scope = commits.reachable
-                     .since(active_task.since_commit)
-                     .until(active_task.until_commit)
-
-      yield scope if block_given?
-
-      scope.select(&:active?)
-           .map.with_index { |c, i| UndeployedCommit.new(c, index: i) }
-           .reverse
-    end
+    def next_expected_commit_to_deploy(commits: nil)
```

Comment:
I see no unit tests for this - should we add some?

Reply:
Yes, I will add some, thanks!